standard/docs/en/schema/reference.md

```diff
@@ -480,7 +482,7 @@ As well as providing this machine-readable link between processes, publishers ma
 
 ### Location
 
-The addition of location information is currently handled through a [proposed extension](../extensions/location.md) to the standard.
+The optional [Location](http://extensions.open-contracting.org/en/extensions/location/) extension can be used to provide location information.
```

Comment:
Should this be a link to the correct version of the extension for this OCDS release? (the current link redirects to the master branch)

Reply:
Done in 127b5d4